src/components/SettingsSelectGroup/SettingsSelectGroup.vue

```diff
@@ -0,0 +1,121 @@
+<!--
+  - @copyright Copyright (c) 2019 Julius Härtl <jus@bitgrid.net>
+  -
+  - @author Julius Härtl <jus@bitgrid.net>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+
+<template>
+	<Multiselect :value="inputValue"
+		:options="groupsArray" :options-limit="5"
+		:placeholder="label"
+		track-by="id"
+		label="displayname"
+		class="multiselect-vue" :multiple="true"
+		:close-on-select="false" :tag-width="60"
+		:disabled="disabled" @input="update"
+		@search-change="asyncFindGroup">
+		<span slot="noResult">{{ t('settings', 'No results') }}</span>
+	</Multiselect>
+</template>
+
+<script>
+import axios from 'nextcloud-axios'
+import Multiselect from 'nextcloud-vue/dist/Components/Multiselect'
+let uuid = 0
```

Comment:
this looks suspicious. is there any way around this module-global variable? This will be shared across all instances of the component if I'm not mistaken.

Reply:
Yeah, we have an uuid generator in the vue components for this purpose exactly :)

Reply:
yes, this one right: https://github.com/nextcloud/nextcloud-vue/blob/master/src/utils/GenRandomId.js

will update asap